espnet2/train/spk_trainer.py

```diff
@@ -0,0 +1,185 @@
+"""Trainer module for speaker recognition."""
```

Comment:
ditto
add a test

Reply:
This one' already included in ci/test_integration2.sh! :)